### PR TITLE
Lagom: Build with -Wno-literal-suffix when using GCC

### PR DESCRIPTION
**Pull request by linusg**

### CODE REVIEW

File: Meta/Lagom/CMakeLists.txt [hunk 1]
```diff
@@ -49,17 +47,14 @@ file(GLOB LIBREGEX_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibRegex/
 file(GLOB LIBCORE_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibCore/*.cpp")
 file(GLOB LIBELF_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibELF/*.cpp")
 # There's no way we can reliably make this cross platform
-list(REMOVE_ITEM LIBELF_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/../../Userland/Libraries/LibELF/DynamicLinker.cpp")
-
-
+list(REMOVE_ITEM LIBELF_SOURCES "../../Userland/Libraries/LibELF/DynamicLinker.cpp")
```
Comment (alimpfard) [inline Meta/Lagom/CMakeLists.txt:50]: This won't remove the entry, file(GLOB) prepends the current directory to the resolved path.

Comment (linusg) [reply Meta/Lagom/CMakeLists.txt]: Fixed, ty!

Comment (linusg) [reply Meta/Lagom/CMakeLists.txt]: Added `CMAKE_CURRENT_SOURCE_DIR` to the other `REMOVE_ITEM` thing instead, no idea how that ever worked - maybe it didn't...

**Comment by bgianfo** [comment]
Whoops, sorry about that. Thanks for the fix!

**Pull request merged**